scripts/ps/create_default_config.ps1

```diff
@@ -1,6 +1,6 @@
 $repoDir = Join-Path $PSScriptRoot ".." | Join-Path -ChildPath ".."
 if (-not (Join-Path $repoDir config.toml | Test-Path)) {
-    $example = Join-Path $repoDir config.example.toml
+    $example = Join-Path $repoDir config.test.toml
```

Comment:
there are few differences in example and test:
timeout (each) in test is set to 1, it is definitely too low for test env.